stage #0/module #4 Algorithms, Data Structures/lesson #05 Data structures (Linear)/README.md

```diff
@@ -0,0 +1,195 @@
+<h1 align="center">Data structures (Linear)</h1>
+
+It is a type of data structure where the arrangement of the data follows a linear trend. The data elements are arranged linearly such that the element is directly linked to its previous and the next elements. As the elements are stored linearly, the structure supports single-level storage of data. And hence, traversal of the data is achieved through a single run only.
+
+## Array
+An array is a structure of fixed-size, which can hold items of the same data type. It can be an array of integers, an array of floating-point numbers, an array of strings or even an array of arrays (such as 2-dimensional arrays). Arrays are indexed, meaning that random access is possible.
```

Comment:
The same comment as [here](https://github.com/mjc-school/MJC-School/pull/74/files#r811828217).

Original source: https://towardsdatascience.com/8-common-data-structures-every-programmer-must-know-171acf6a1a42